service/src/test/java/io/camunda/service/license/CamundaLicenseTest.java

```diff
@@ -108,7 +107,7 @@ public void shouldReturnProperValidValuesWhenLicenseIsSaas() throws InvalidLicen
     testLicense.initializeWithLicense("whatever");
 
     // then
-    assertFalse(testLicense.isSelfManaged());
+    assertEquals("saas", testLicense.getLicenseType());
```

Comment:
❓ Can we store those license type values somewhere ? I'd prefer having them in one place instead of spread those string everywhere. Wdyt ? (See also the suggestion above)